src/styles/utils/_extends.scss

```diff
@@ -0,0 +1,14 @@
+%flexCenterXY {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+%handsPositioning {
+  position: absolute;
+}
```

Comment:
I recommend you make it a mixin that can calculate the `top` offset for hands. So you wouldn't need to use so strange solution with transform.